### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by garyghayrat**

### CODE REVIEW

File: pages/contracts/erc5564-contract.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+The `ERC5564Announcer` contract is designed to facilitate the broadcasting of transactions involving stealth addresses by emitting an `Announcement` event. This event contains relevant information that can be used to reconstruct the transaction and validate its parameters.
+
+### Code: [`ERC5564Announcer.sol`](https://github.com/ScopeLift/stealth-address-erc-contracts/blob/b6f7d989b6247f81ceede5d1be2ffb23c1cb39b5/src/ERC5564Announcer.sol)
+
+## Events
+
+### Announcement
+
+The `Announcement` event is emitted to when the `announce` function is called. Here are the key components of this event:
```
Comment (marcomariscal) [inline pages/contracts/erc5564-contract.mdx]:
```suggestion
The `Announcement` event is emitted when the `announce` function is called. Here are the key components of this event:
```

File: pages/contracts/erc5564-contract.mdx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+```solidity
+function announce(
+  uint256 schemeId,
+  address stealthAddress,
+  bytes memory ephemeralPubKey,
+  bytes memory metadata
+) external
+```
+
+### Description
```
Comment (marcomariscal) [inline pages/contracts/erc5564-contract.mdx]:
```suggestion
#### Description
```

File: pages/contracts/erc6538-contract.mdx [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+### Code: [`ERC6538Registry.sol`](https://github.com/ScopeLift/stealth-address-erc-contracts/blob/b6f7d989b6247f81ceede5d1be2ffb23c1cb39b5/src/ERC6538Registry.sol)
+
+## Events
+
+### StealthMetaAddressSet
+
+The `StealthMetaAddressSet` event is emitted when a registrant updates their stealth meta-address.
+
+- **registrant** (`address`): The account that the registered the stealth meta-address belongs to.
```
Comment (marcomariscal) [inline pages/contracts/erc6538-contract.mdx]:
```suggestion
- **registrant** (`address`): The account the registered stealth meta-address belongs to.
```

File: pages/contracts/erc6538-contract.mdx [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+#### Event Signature
+
+```solidity
+event StealthMetaAddressSet(
+  address indexed registrant, uint256 indexed schemeId, bytes stealthMetaAddress
+);
+```
+
+### NonceIncremented
```
Comment (marcomariscal) [inline pages/contracts/erc6538-contract.mdx]: maybe events, errors, and funcs can be `code` to differentiate between other headers

Comment (marcomariscal) [reply pages/contracts/erc6538-contract.mdx]:
```suggestion
### `NonceIncremented`
```

**Review by marcomariscal — APPROVED** [review]
looks beautiful; just a couple of nits

File: pages/contracts/erc6538-contract.mdx (hunk 4 above)
Comment (garyghayrat) [reply pages/contracts/erc6538-contract.mdx]: Fixed in https://github.com/ScopeLift/stealth-address-docs/pull/12/commits/f98ca208f1f49f9768386581c15f682ab4fc877a

File: pages/contracts/erc6538-contract.mdx (hunk 3 above)
Comment (garyghayrat) [reply pages/contracts/erc6538-contract.mdx]: https://github.com/ScopeLift/stealth-address-docs/pull/12/commits/bf7ffddbc12a7e0869b175001060935f6e7eebc4

File: pages/contracts/erc5564-contract.mdx (hunk 2 above)
Comment (garyghayrat) [reply pages/contracts/erc5564-contract.mdx]: https://github.com/ScopeLift/stealth-address-docs/pull/12/commits/bf7ffddbc12a7e0869b175001060935f6e7eebc4

File: pages/contracts/erc5564-contract.mdx (hunk 1 above)
Comment (garyghayrat) [reply pages/contracts/erc5564-contract.mdx]: https://github.com/ScopeLift/stealth-address-docs/pull/12/commits/bf7ffddbc12a7e0869b175001060935f6e7eebc4

**Pull request merged**